configure-buildfarm/defaults/main.yml

```diff
@@ -1,6 +1,23 @@
 ---
 
-# The platfroms that are supported by the self-managed build farm. 
+# The platfroms that are supported by the self-managed build farm.
 # Valid platforms are 'android' and 'ios'. Multiple values should be separated by comma
 project_name: jenkins
 buildfarm_version: "2.0"
+
+jenkins_home: /var/lib/jenkins
+jenkins_plugins:
+  -
+    name: "ssh-slaves"
+    version: 1.20
+    archive: "http://updates.jenkins-ci.org/download/plugins/ssh-slaves/1.20/ssh-slaves.hpi"
```

Comment:
Yes, these values are coming from role defaults file which takes least precedence of all variables and so can be overridden anywhere